src/main/views/courts/tabs/typesContent.njk

```diff
@@ -4,7 +4,7 @@
 {% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
 {% from "govuk/components/panel/macro.njk" import govukPanel %}
 
-<h2 class="govuk-heading-l">Court Types and Codes</h2>
+<h2 class="govuk-heading-l">Types and Codes</h2>
```

Comment:
Isn't the previous one 'court types and codes' sounds a bit better? This is the main heading on that page and 'types and codes' just feel a bit too generic here.

Reply:
agreed changed it back